master-firmware/src/strategy.cpp

```diff
@@ -548,6 +549,142 @@ struct BuildTowerLevel : actions::BuildTowerLevel {
     }
 };
 
+void strat_collect_wastewater(enum strat_color_t color)
+{
+    ballgun_tidy(&main_ballgun);
+    ballgun_deploy(&main_ballgun);
+    strategy_wait_ms(1000);
```

Comment:
Thats a lot of delay. Is it really needed ?

Reply:
Not tuned really